pkg/sql/parser/sql.y

```diff
@@ -844,7 +844,7 @@ func (u *sqlSymUnion) setVar() *tree.SetVar {
 %token <str> START STATISTICS STATUS STDIN STREAM STRICT STRING STORAGE STORE STORED STORING SUBSTRING
```

Comment:
you'll probably want to add some parsing tests in `pkg/sql/parser/testdata/` (`make test PKG=./pkg/sql/parser TESTS=TestParse TESTFLAGS='-rewrite'` to rewrite)